docs/agentic-1813/06-advanced-security-and-monitoring.md

```diff
@@ -65,15 +65,18 @@ Now customize the input values for your Code Engine DA:
 1. On the Edit agentic-ai-demo page, scroll down and click **Next** to continue.
 1. To enter the **API key**, simply copy and paste the value of the key you created earlier.
 1. Click **Next** to continue.
-1. Enter your Watsonx API key into the **watsonx_ai_api_key** field. **This step is optional** — if the key is not provided, the IBM Cloud API key will be used by default.
-1. We will link the input of the `agentic-ai-demo` to the output of the previously deployed `watsonx-ai-saas-demo` DA. Hover over the **watsonx_project_id** input field and click on **Add Reference**.
+1. We will link the input of the `agentic-ai-demo` to the output of the previously deployed `watsonx-ai-saas-demo` DA. Hover over the **watsonx_project_id** input field and click on **Add a reference**.
 1. A new popup will open.
     - For **Source**, select `Configuration`.
     - For **Name**, choose your `watsonx-ai-saas-demo`.
     - Set **Category** to `Output`.
     - For **Property**, select `watsonx_project_id`.
-1. Click **OK** to confirm the reference selection.
-1. Click the **Optional inputs** toggle to display optional input variables, then modify the **prefix** value as needed.
+    - Click **OK** to confirm the reference selection.
+1. In the input field labeled `prefix`, enter the `<your-initials>-txc-demo`.
```

Comment:
prefix, is not under optional section, I updated this part to reflect real scenario